### PR TITLE
fix: update eslint to latest version

### PR DESCRIPTION
**Pull request by evenstensberg**
**What**:
See https://github.com/webpack/webpack/issues/8656

It breaks webpack builds.

**How**:

Npm install

### CODE REVIEW

**Comment by bj00rn** [comment]
#144 should be the preferred approach (IMHO)

**Comment by evenstensberg** [comment]
Do you mind merging that when possible? We’re getting a lot of complaints about that

**Comment by zimme** [comment]
As you can see this has failing tests and that's most likely because we can't just bump to a new major version of eslint without changing the way we use the functionality of eslint in this package as well as change the way `prettier-eslint` uses eslint and bump the version of eslint in that package too..

However, for me to being able to publish any new releases prettier-eslint I'll need to fix the problem with mocking the filesystem which has stopped working because of an updated in prettier... So this and a lot of other issues will be merged and fixed only after I or someone find a way to fix the filesystem mocking.

**Comment by evenstensberg** [comment]
Gotcha

**Comment by bj00rn** [comment]
@zimme @evenstensberg the reason this PR fails is resolved in #150

File: package.json [hunk 1]
```diff
@@ -27,7 +30,7 @@
     "camelcase-keys": "^4.1.0",
     "chalk": "2.3.0",
     "common-tags": "^1.4.0",
-    "eslint": "^4.5.0",
+    "eslint": "5.13.0",
```
Comment (chinesedfan) [inline package.json]: How about using range?

**Comment by zimme** [comment]
Fixed in version 5.0.0

**Pull request closed without merging**